tests/integration/Core/Framework/Store/Services/ExtensionLoaderTest.php

```diff
@@ -128,6 +129,51 @@ public function testItLoadsExtensionsFromPlugins(): void
 
         static::assertNotNull($extension);
         static::assertEquals('AppStoreTestPlugin', $extension->getName());
+        static::assertTrue($extension->isAllowUpdate());
+
+        $pluginId = $extension->getLocalId();
+        static::assertNotNull($pluginId);
+
+        $pluginRepo->upsert([
+            [
+                'id' => $pluginId,
+                'managedByComposer' => true,
+            ],
+        ], Context::createDefaultContext());
+
+        /** @var PluginCollection $plugins */
```

Comment:
please do not use var annotation in tests. could all be replaced by proper assertions